src/main/kotlin/com/autonomousapps/model/Coordinates.kt

```diff
@@ -190,6 +190,25 @@ data class ModuleCoordinates(
   override val gradleVariantIdentification: GradleVariantIdentification,
 ) : Coordinates(identifier, gradleVariantIdentification) {
   override fun gav(): String = "$identifier:$resolvedVersion"
+
+  internal fun toVersionCatalogAlias(): String {
+    return "${identifier}-${resolvedVersion}"
+      .split(':', '.')
+      // replace reserved keywords with safe alternatives
+      .joinToString(separator = "-") { tomlReservedKeywordMappings.getOrDefault(it, it) }
+      .lowercase()
+  }
```

Comment:
Can you add some unit tests in `CoordinatesTest` for this functionality?

Hmm... do we need to include the `resolvedVersion` in the ailas for the case where there are duplicates (except by version)? Would be nice if we had the version suffix only when there were duplicates and not always?